webapp/backend/.gitignore

```diff
@@ -0,0 +1,8 @@
+// node modules
+node_modules
+
+// environment variables
+.env
+
+// Output files
+build
```

Comment:
add a newline at the end of the file to remove the mark here. if you hover over it you'll see what the issue is as well

more details on it here https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
@wsedor Thank you for the link. Newline added at the EOF.